autopilot/contractor/contractor.go

```diff
@@ -544,149 +518,6 @@ func canSkipContractMaintenance(ctx context.Context, cfg api.ContractsConfig) (s
 	return "", false
 }
 
-func computeContractSetChanged(ctx *mCtx, alerter alerts.Alerter, bus Bus, churn *accumulatedChurn, logger *zap.SugaredLogger, oldSet, newSet []api.ContractMetadata, toStopUsing map[types.FileContractID]string) (bool, error) {
```

Comment:
Yeah I wanted to have that discussion here and follow up with that actually. I was thinking perhaps a simple alert that lists what contracts were marked as bad and why is enough. I eventually opted remove the churn alert, the concept of churn ended up feeling too closely related to contract sets. I was also thinking that instead of an alert, perhaps we should this information in the contracts table instead. If that is the case I think we don't even need an alert at all and reserve those for when the user has to take action, e.g. not enough contracts, not enough hosts and so on.

Reply:
Imo the simpler alert is a bit too much of a step back in terms of UX. At the very least I'd expect a number indicating how many times the contract has flipped since `renterd`s startup. Churn can still be an issue without contract sets.

Reply:
I updated the PR to include a timeline of usability changes per contract. I think this should be sufficient for Alex to rebuild the UI to something that closely matches what we had before. I have not play tested it yet, I would like to merge the two open PRs into the base branch and then test everything using the new UI. If you approve of the new alert, would you mind pinging Alex so he can update the UI accordingly?